doc/man/scons.xml

```diff
@@ -102,35 +102,49 @@ orchestrates the construction of software
 (and other tangible products such as documentation files)
 by determining which
 component pieces must be built or rebuilt and invoking the necessary
-commands to build them.</para>
+commands to build them.
+&SCons; offers many features to improve developer productivity
+such as parallel builds, caching of build artefacts, and
+a database of information about previous builds so details do
+not have to be recalculated each run.
+</para>
 
-<para>You instruct
-&scons; by writing a configuration file
-which specifies the files to be built (<firstterm>targets</firstterm>),
-and, if necessary, the rules to build those files.  Premade
-rules exist for building many common software components
+<para>You set up an &SCons;
```

Comment:
a SCons ?

Reply:
SCons starts with a "vowel sound" (ess-cons), so "an" is correct.